test/integration/features/addon-registry-route.feature

```diff
@@ -43,6 +43,20 @@ so that user can able to push their container images directly to registry and de
       And with up to "10" retries with wait period of "500ms" the "status code" of HTTP request to "/" of service "nginx" in namespace "myproject" is equal to "200"
       And with up to "10" retries with wait period of "500ms" the "body" of HTTP request to "/" of service "nginx" in namespace "myproject" contains "Welcome to nginx!"
 
+  Scenario: As a user, I can login to OpenShift integrated docker registry after restart minishift
+    Given Minishift has state "Running"
+     When executing "minishift stop" succeeds
+     Then executing "minishift start --iso-url centos" succeeds
+      And Minishift should have state "Running"
+      And executing "minishift docker-env" in host shell succeeds
```

Comment:
Remove the duplicate lines from 51 to 54 as host shell is already configured in the previous scenario